### PR TITLE
Lambda Layer Update - Documentation for October release

### PR DESCRIPTION
**Pull request by NathanielRN**
**Description:**

Update documentation to mention the latest layers released in ADOT October release.

Note: Should be merged with aws-otel/aws-otel.github.io#196

**Link to tracking Issue:**

N/A

**Testing:**

N/A

**Documentation:**

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 - Python3.8 layer [aws-otel-python38-ver-1-5-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.5.0` with the AWS Python Extension `v1.0.1`
-- Nodejs layer [aws-otel-nodejs-ver-0-25-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript `v0.25.0` with AWS Lambda Instrumentation `v0.26.0`
-- Java-Wrapper layer [aws-otel-java-wrapper-ver-1-6-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.6.0`
-- Java-Agent layer [aws-otel-java-agent-ver-1-6-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS OpenTelemetry Java instrumentation `v1.6.0`
-- Collector layer [aws-otel-collector-ver-0-36-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) contains ADOT Collector for Lambda `v0.13.0`
+- Nodejs layer [aws-otel-nodejs-ver-1-0-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.0.0`, OpenTelemetry Contrib `v0.26.0`, and AWS Lambda Instrumentation `v0.1.0`
```
Comment (willarmiros) [inline README.md:5]:
```suggestion
- Nodejs layer [aws-otel-nodejs-ver-1-0-0](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.0.0` with AWS Lambda Instrumentation `v0.27.0`
```

OTel JS contrib has moved to completely independent releases for each package, so there is no longer a concept of a "contrib version". Also, the lambda instrumentation version used is 0.27.0: https://github.com/open-telemetry/opentelemetry-lambda/blob/2921a4582fd0796585195a7cebdce74458342b22/nodejs/packages/layer/package.json#L32

**Review by willarmiros — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NathanielRN) [reply README.md:5]: Ah awesome, thank you for this! I thought this was the "AWS Instrumentation" for some reason... made the change! 🙂

**Review by willarmiros — APPROVED** [review]

**Comment by pdecat** [comment]
Wasn't there a new release of the Python layer too?

FWIW, I'm looking for a release that includes https://github.com/aws-observability/aws-otel-lambda/pull/155 which is supposed to resolve https://github.com/aws-observability/aws-otel-lambda/issues/124 which I'm randomly facing nowadays.

**Comment by willarmiros** [comment]
@pdecat unfortunately we skipped including the Python layer in the latest release because there was a regression with the OTel -> X-Ray data model translation. It is now resolved and we hope to get it out in a release soon!

**Comment by pdecat** [comment]
Hi @willarmiros, thanks for the quick feedback!

Any idea if #155 actually does resolve #124?

**Comment by willarmiros** [comment]
Maybe @NathanielRN would know, but it seems like it probably doesn't directly resolve your issue, rather just makes debugging it easier by allowing for more configuration.

**Comment by NathanielRN** [comment]
Hi @pdecat ! Hopefully we can help answer your question.

Unfortunately I don't believe #155 will resolve #124. The goal of #155 was to add the `tracecontext` and `b3` propagators because that [is what the upstream OTel Python does by default](https://github.com/open-telemetry/opentelemetry-python/blob/1f85268a6f00de4a771e0aaa8663ca6fbc07c575/opentelemetry-api/src/opentelemetry/propagate/__init__.py#L128-L132). This just allows OTel Python to parse out a "tracing context" from the incoming request if it is available. But by default an SQS/Eventbrigde triggered Lambda won't have a `tracecontext` or `b3` readable "OTel Context". That means these propagators are normally just a "no-op".

**Pull request merged**